docs/reference/search/search-your-data.asciidoc

```diff
@@ -227,9 +177,45 @@ GET /*/_search
 ----
 // TEST[setup:my_index]
 
-include::request/index-boost.asciidoc[]
-include::request/preference.asciidoc[]
-include::request/search-type.asciidoc[]
+[discrete]
+[[common-search-options]]
+=== Common search options
+
+You can use the following options to customize your searches.
```

Comment:
Should we briefly link to the query DSL and aggregation docs here? It seems like a natural first thing to wonder ("how do I actually specify the search, beyond the example given above?")